sonar/modules/permissions.py

```diff
@@ -28,6 +28,8 @@
 superuser_access_permission = Permission(ActionNeed('superuser-access'))
 admin_access_permission = Permission(RoleNeed('librarian'), RoleNeed('admin'))
 moderator_access_permission = Permission(ActionNeed('admin-access'))
+user_access_permission = Permission(RoleNeed('user'),
+                                    RoleNeed('librarian'), RoleNeed('admin'))
```

Comment:
Is librarian make sense for SONAR? I can suggest moderator instead. user seems also confusing as it can be interpreted as a person that has an account, perhaps you can use submitter...

Reply:
absolutely correct, it's a mistake, it's moderator, not librarian.. Corrected.